.config/nextest.toml

```diff
@@ -14,3 +14,7 @@ retries = 3
 status-level = "skip"
 failure-output = "immediate-final"
 junit = { path = "junit.xml" }
+
+[profile.default.slow-timeout]
```

Comment:
Will this be used for all other profiles, too? e.g., `ci` and `smoke-coverage`? I'm just wondering if we've verified that it works for unit tests and smoke tests?

Reply:
It's being applied immediately to the branch being tested, so I know this passes for rust-unit-tests. I don't know when smoke tests are run (none of my PRs seem to cause it).